task-2/package.json

```diff
@@ -16,10 +16,15 @@
 	"author": "Mohammed Raheem",
 	"license": "MIT",
 	"dependencies": {
+		"@reduxjs/toolkit": "^1.7.1",
 		"prop-types": "^15.8.1",
 		"react": "^17.0.2",
 		"react-datepicker": "^4.6.0",
-		"react-dom": "^17.0.2"
+		"react-dom": "^17.0.2",
+		"react-redux": "^7.2.6",
+		"redux": "^4.1.2",
+		"redux-devtools-extension": "^2.13.9",
```

Comment:
This package is not needed and is deprecated.

Reply:
Removed!